app/src/pages/inside/projectSettingsPageContainer/content/patternAnalysis/patternAnalysisContent/patternAnalysisContent.jsx

```diff
@@ -158,7 +160,13 @@ export const PatternAnalysisContent = ({
 
   return (
     <>
-      <Layout description={formatMessage(messages.tabDescription)}>
+      <Layout
+        description={Parser(
+          formatMessage(messages.tabDescription, {
+            a: (data) => createLink(data, 'https://reportportal.io/docs/analysis/PatternAnalysis'),
```

Comment:
We have a `src/common/utils/referenceDictionary.js` file to store RP related resources.
As the same links can be used across our UI app and to simplify they editing in the future I suggest to move all the links to the reference dictionary. It may be one more object in the file to store Docs related links.